telegram/ext/conversationhandler.py

```diff
@@ -136,6 +136,11 @@ class ConversationHandler(Handler[Update]):
         map_to_parent (Dict[:obj:`object`, :obj:`object`], optional): A :obj:`dict` that can be
             used to instruct a nested conversationhandler to transition into a mapped state on
             its parent conversationhandler in place of a specified nested state.
+        run_async (:obj:`bool`, optional): Pass :obj:`True` to override the
+            :attr:`Handler.run_async` setting of all handlers (in :attr:`entry_points`,
+            :attr:`states` and :attr:`fallbacks`.
```

Comment:
```suggestion
        run_async (:obj:`bool`, optional): Pass :obj:`True` to override the
            :attr:`Handler.run_async` setting of all handlers.
```

We try to keep the extensive documentation to the Parameters and have shorter ones which are sufficient in the attributes :)